### PR TITLE
remove antlr4-python3-runtime for 1.7.8

### PR DESCRIPTION
**Pull request by suzhoum**
Fixes https://github.com/conda-forge/statsforecast-feedstock/issues/43 for 1.7.8

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by suzhoum** [comment]
It might need to be merged to `master` for the workflow to publish to conda-forge. Maybe we could revert it to the current master after this fix @jmoralez?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 {% set build = 1 %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/statsforecast-{{ version }}.tar.gz
-  sha256: 7bd7145c1b0abeb672c9d7334d3f8eb51d28c9ce61465a3321d375c3f099c6b9
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/statsforecast-{{ version }}.tar.gz
```
Comment (jmoralez) [inline recipe/meta.yaml:10]:
```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/statsforecast-{{ version }}.tar.gz
```

Comment (suzhoum) [reply recipe/meta.yaml]: Updated!

**Comment by jmoralez** [comment]
I'm not sure if we have to revert master after this, the bot updates the version and the hash when it detects an update on PyPI, I guess it'll just bump back to 2.0.1 or whatever our next release is.

**Review by jmoralez — APPROVED** [review]

**Comment by suzhoum** [comment]
Please feel free to merge when you feel appropriate. Thank you for your assistance!

**Comment by jmoralez** [comment]
Thanks for the fix!

**Pull request merged**